bindata/bootstrap/cloudcredential_v1_operator_config_custresdef.yaml

```diff
@@ -4,6 +4,7 @@ metadata:
   name: cloudcredentials.operator.openshift.io
   annotations:
     include.release.openshift.io/self-managed-high-availability: "true"
+    include.release.openshift.io/single-node-developer: "true"
```

Comment:
Updating this through `make update-vendored-crds` was required to pass `make verify`